lib/datasets/zip_writer.rb

```diff
@@ -10,45 +11,40 @@ class ZipWriter
   # @yield [input_zip, output_zip] optional block for implementation of copying
   # from input zip to output zip. By default gets all txt files in input zip
   # and copies them to output zip.
-  def write(src_path, dest_path)
-    # open the source and destination zips
-
-    temp_zip = Tempfile.open(['dataset', '.zip'], File.dirname(dest_path))
-
-    begin
-      # initialize zip file - required since we opened the file w/ Tempfile
-      Zip::OutputStream.open(temp_zip) { |zos| }
```

Comment:
I tried removing this to see if doing so would break the tests, but they still passed.